### ISSUE TITLE
From distogram to structure?

### ISSUE THREAD

**Issue by karchern** [deepmind/deepmind-research#43]
Disclaimer: I'm not a structural biologist and I'm not working in protein structure prediction. I can't say I've understood all the details here.

I've read the the AlphaFold publication and was so intrigued by it that I wanted to predict the structure a couple of proteins that we typically look at (here in the lab) just out of curiosity.

From the README, I can see that it is possible to recreate the CASP13 results (distograms etc.).

My question: Is it possible to get from input protein sequence to predicted structure using AlphaFold?

**Comment by Augustin-Zidek** [comment]
Hi,

I recommend taking a look at https://github.com/deepmind/deepmind-research/issues/28 which discusses this topic.

TL;DR: This is unfortunately not possible as we didn't open source the feature generation code. (It's tightly coupled to our internal infrastructure as well as external tools which we cannot open source.)

**Comment by karchern** [comment]
Hi,

that is really a shame! Anyway, thanks for your answer :)
